solidity-plugin/src/main/java/org/sonarsource/solidity/MyVisitor.java

```diff
@@ -0,0 +1,36 @@
+package org.sonarsource.solidity;
+
+import org.antlr.v4.runtime.Token;
+import org.antlr.v4.runtime.TokenStream;
+import org.sonar.api.batch.sensor.highlighting.NewHighlighting;
+import org.sonar.api.batch.sensor.highlighting.TypeOfText;
+import org.sonarsource.solidity.frontend.SolidityBaseVisitor;
+
+public class MyVisitor extends SolidityBaseVisitor<Token> {
+
+  private NewHighlighting highlighting;
+
+  public MyVisitor(NewHighlighting highlighting) {
+    this.highlighting = highlighting;
+  }
+
+  public void visitTokens(TokenStream tokenStream) {
+    for (int i = 0; i < tokenStream.size(); i++) {
+      Token token = tokenStream.get(i);
+      if (SoliditySensor.KEYWORDS.contains(token.getText())) {
+        this.highlighting.highlight(token.getLine(), token.getCharPositionInLine(),
+          token.getLine(), (token.getCharPositionInLine() + token.getStopIndex() - token.getStartIndex() + 1), TypeOfText.KEYWORD);
+      } else if (SoliditySensor.KEYWORD_TYPES.contains(token.getText())) {
+        this.highlighting.highlight(token.getLine(), token.getCharPositionInLine(),
+          token.getLine(), (token.getCharPositionInLine() + token.getStopIndex() - token.getStartIndex() + 1), TypeOfText.KEYWORD_LIGHT);
+      } else if (token.getType() == 115) {
+        this.highlighting.highlight(token.getLine(), token.getCharPositionInLine(),
+          token.getLine(), (token.getCharPositionInLine() + token.getStopIndex() - token.getStartIndex() + 1), TypeOfText.STRING);
+      } else if (token.getType() >= 96 && token.getType() <= 100) {
+        this.highlighting.highlight(token.getLine(), token.getCharPositionInLine(),
+          token.getLine(), (token.getCharPositionInLine() + token.getStopIndex() - token.getStartIndex() + 1), TypeOfText.CONSTANT);
+      } else {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Either remove or fill this block of code. ([squid:S00108](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3AS00108))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=14&issues=AWQCnzhfuoX1TGVkLScq&open=AWQCnzhfuoX1TGVkLScq)

[ISSUE_KEY]: AWQCnzhfuoX1TGVkLScq